recipe/meta.yaml

```diff
@@ -1,15 +1,19 @@
 {% set name = "arraylake" %}
-{% set version = "0.12.0" %}
+{% set version = "0.13.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/arraylake-{{ version }}.tar.gz
-  sha256: ad6a846f9358b1e0e15daa389fda2fb0abb5da536640034a79da9f5d4da6bcb5
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/arraylake-{{ version }}.tar.gz
+  sha256: bb67bd1d8e2a431cd8ef70816201c940d546ad229821cc8fffae5f3a7d2c1566
 
 build:
+  # https://hatch.pypa.io/1.12/config/build/#environment-variables
+  # disable hatch-min-requirements, it is not available on conda-forge yet
+  script_env:
+    - HATCH_BUILD_HOOK_ENABLE_MIN_REQUIREMENTS=false
```

Comment:
Apparently I haven't set the env var correctly:

```
hatchling.plugin.exceptions.UnknownPluginError: Unknown metadata hook: min_requirements
```

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1092217&view=logs&j=7b6f2c87-f3a7-5133-8d84-7c03a75d9dfc&t=9eb77fd2-8ddd-5444-8fc0-71cb28dcb736&l=812

Reply:
ah this disables a "build" hook but we want to disable a "metadata" hook. I don't see how to do this.